atd-toolbox/data_model/python/sql_templates/crashes_list_view.sql

```diff
@@ -140,10 +261,46 @@ create or replace view crash_injury_metrics_view as
         coalesce(
             sum(person_injury_metrics_view.cris_fatal_injury), 0
         ) as cris_fatality_count,
+        coalesce(sum(
+            person_injury_metrics_view.motor_vehicle_fatal_injry
+        ), 0) as motor_vehicle_fatality_count,
+        coalesce(sum(
+            person_injury_metrics_view.motor_vehicle_sus_serious_injry
+        ), 0) as motor_vehicle_sus_serious_injry_count,
+        coalesce(sum(
+            person_injury_metrics_view.motorcycle_fatal_injry
+        ), 0) as motorcycle_fatality_count,
+        coalesce(sum(
+            person_injury_metrics_view.motorycle_sus_serious_injry
+        ), 0) as motorcycle_sus_serious_count,
+        coalesce(sum(
+            person_injury_metrics_view.bicycle_fatal_injry
+        ), 0) as bicycle_fatality_count,
+        coalesce(sum(
+            person_injury_metrics_view.bicycle_sus_serious_injry
+        ), 0) as bicycle_sus_serious_injry_count,
+        coalesce(sum(
+            person_injury_metrics_view.pedestrian_fatal_injry
+        ), 0) as pedestrian_fatality_count,
+        coalesce(sum(
+            person_injury_metrics_view.pedestrian_sus_serious_injry
+        ), 0) as pedestrian_sus_serious_injry_count,
+        coalesce(sum(
+            person_injury_metrics_view.micromobility_fatal_injry
+        ), 0) as micromobility_fatality_count,
+        coalesce(sum(
+            person_injury_metrics_view.micromobility_sus_serious_injry
+        ), 0) as micromobility_sus_serious_injry_count,
+        coalesce(sum(
+            person_injury_metrics_view.other_fatal_injry
+        ), 0) as other_fatality_count,
+        coalesce(sum(
+            person_injury_metrics_view.other_sus_serious_injry
+        ), 0) as other_sus_serious_injry_count,
         case
             when (sum(person_injury_metrics_view.fatal_injury) > 0) then 4
-            when (sum(person_injury_metrics_view.nonincap_injry) > 0) then 1
-            when (sum(person_injury_metrics_view.sus_serious_injry) > 0) then 2
+            when (sum(person_injury_metrics_view.nonincap_injry) > 0) then 2
+            when (sum(person_injury_metrics_view.sus_serious_injry) > 0) then 1
```

Comment:
dont we want the `crash_injry_sev_id` to be representative of the most severe injury involved in the crash? so fatality -> serious injury -> nonincap injury etc? or am i misunderstanding this field